### ISSUE TITLE
Multi Agent Navigation in Unconstrained Environments using a Centralized Attention based Graphical Neural Network Controller. (arXiv:2307.16727v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27237]
Multi Agent Navigation in Unconstrained Environments using a Centralized Attention based Graphical Neural Network Controller. (arXiv:2307.16727v2 [cs.RO] UPDATED)<br>
https://ift.tt/3WQ9qX7<br>
<p>In this work, we propose a learning based neural model that provides both the
longitudinal and lateral control commands to simultaneously navigate multiple
vehicles. The goal is to ensure that each vehicle reaches a desired target
state without colliding with any other vehicle or obstacle in an unconstrained
environment. The model utilizes an attention based Graphical Neural Network
paradigm that takes into consideration the state of all the surrounding
vehicles to make an informed decision. This allows each vehicle to smoothly
reach its destination while also evading collision with the other agents. The
data and corresponding labels for training such a network is obtained using an
optimization based procedure. Experimental results demonstrates that our model
is powerful enough to generalize even to situations with more vehicles than in
the training data. Our method also outperforms comparable graphical neural
network architectures. Project page which includes the code and supplementary
information can be found at https://ift.tt/nTPFl4e
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
